.editorconfig

```
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = tab
```

Maybe we can use soft spaces instead?

```suggestion
indent_style = space
indent_size = 4
```
I noticed some spacing inconsistencies.